dali/operators/decoder/image_decoder.cc

```diff
@@ -147,31 +182,35 @@ Supported formats: JPG, BMP, PNG, TIFF, PNM, PPM, PGM, PBM.)code")
 
 
 DALI_SCHEMA(ImageDecoderSlice)
-  .DocStr(R"code(Decode images and extract an externally provided region-of-interest (ROI) specified
-by an anchor and a shape of the ROI.
-
-Inputs must be supplied as 3 separate tensors in a specific order: `data`
-containing input data, `anchor` containing either normalized or absolute coordinates
-(depending on the value of `normalized_anchor`) for the starting point of the
-slice (x0, x1, x2, ...), and `shape` containing either normalized or absolute coordinates
-(depending on the value of `normalized_shape`) for the dimensions of the slice
-(s0, s1, s2, ...). Both `anchor` and `shape` coordinates must be within the interval
-[0.0, 1.0] for normalized coordinates, or within the image shape for absolute
-coordinates. Both `anchor` and `shape` inputs will provide as many dimensions as specified
-with arguments `axis_names` or `axes`.
-
-By default `ImageDecoderSlice` operator uses normalized coordinates and `WH` order for the slice
-arguments.
-
-When possible, it will make use of region-of-interest decoding APIs (e.g. *libjpeg-turbo*, *nvJPEG*)
-thus optimizing decoding time and memory usage. When not supported, it will decode the whole image
-and then crop the selected ROI.
-
-Note: ROI decoding is currently not compatible with hardware based decoding.
-Using *ImageDecoderSlice* will automatically disable hardware accelerated decoding.
-To make use of the hardware decoder, use *ImageDecoder* and *Slice* operators instead.
+  .DocStr(R"code(Decodes images and extracts a region of interest based on externally provided
+anchors and shapes.
 
-The output of the decoder is in *HWC* layout.
+Inputs must be supplied as tensors in the following order:
+
+* ``data`` that contains the input data.
+* An ``anchor`` that contains normalized or absolute coordinates, depending on the
```

Comment:
```suggestion
* ``anchor`` that contains normalized or absolute coordinates, depending on the
```
It's not "an" anchor - it's a proper name of the argument.

Reply:
Done